docs/guides/support-bundle.md

```diff
@@ -1,5 +1,14 @@
 # Generate and upload a Support Bundle to Coder Support
 
+
```

Comment:
We should probably not add html tags in the markdown and add this information in the markdown header metadata. 
Our docs engine then should parse it. 

See the modules and templates readme to see what I am talking about. 

https://github.com/coder/coder/blob/f21f2dce572083507ec568a80a035a1ceb6f0c50/examples/templates/docker/README.md?plain=1#L1-L8

Reply:
We can also do it as part of docs refactor work.

Reply:
Open to adding this with the restructure. I initially hacked the author info together because we rushed guides. 

Good call.